bigflow/build/__init__.py

```diff
@@ -0,0 +1,20 @@
+__all__ = []
+
+def setup(**kwargs):
+    """Speficy parameters of Bigflow project. Must me called from `setup.py`
+    """
+    # Laziliy import 'bigflow.dist' as it depends on 'setuptools'
+    import bigflow.build.dist as d
+    return d.setup(**kwargs)
+
+
+# TODO: Remove in v2.0. COMPAT
+import bigflow.build.dist as _dist
+from deprecated import deprecated
+for n in [
```

Comment:
Why we use deprecated method here? Why not directly on listed objects?

Reply:
We are still using those functions internally (inside `dist.py`), that is why they are not deprecated with decorator.

Reply:
Ok